react-native-ChartView.js

```diff
@@ -154,6 +153,34 @@ class ChartWeb extends Component {
     };
 };
 
+var flattenObject = function (obj, str='{') {
```

Comment:
Out of curiosity, did you borrow the implementation/idea of `flattenObject` and `toFlatText` from somewhere else or create it yourself?